app/Http/Controllers/UserController.php

```diff
@@ -14,9 +28,32 @@ class UserController extends Controller
     public function index()
     {
         if (!isset($listUsers)) {
```

Comment:
this is a list of users get from database and return to index to show the users record